explorer/src/components/UptimeChart.tsx

```diff
@@ -71,14 +71,16 @@ export const UptimeChart: React.FC<ChartProps> = ({
           }
           options={{
             backgroundColor:
-              mode === 'dark' ? 'rgb(50, 60, 81)' : 'rgb(241, 234, 234)',
+              theme.palette.mode === 'dark'
+                ? theme.palette.nym.networkExplorer.background.tertiary
+                : undefined,
             color: uptimeStory.error
               ? 'rgba(255, 255, 255, 0.4)'
               : 'rgba(255, 255, 255, 1)',
             colors: ['#FB7A21', '#CC808A'],
             legend: {
               textStyle: {
-                color: 'white',
+                color,
```

Comment:
can this be `color: theme.palette.text.primary` instead of declaring the color variable above (line 30)?